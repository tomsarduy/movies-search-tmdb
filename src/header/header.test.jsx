import { shallow } from "enzyme";
import React from "react";
import Header from "./header";

describe("Header", () => {
  it("renders correctly", () => {
    const component = shallow(<Header />);
    expect(component).toMatchSnapshot();
  });
});
