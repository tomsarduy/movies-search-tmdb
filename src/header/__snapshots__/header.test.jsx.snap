// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<header>
  <div>
    <test-file-stub
      fill="black"
      height="100"
    />
    <test-file-stub
      height="100"
    />
  </div>
  <nav>
    <ul>
      <li>
        <a
          _target="blank"
          href="hhttps://www.themoviedb.org/about"
        >
          About TMDb
        </a>
      </li>
      <li>
        <a
          _target="blank"
          href="https://www.themoviedb.org/documentation/api/terms-of-use"
        >
          API Terms of Use
        </a>
      </li>
      <li>
        <a
          _target="blank"
          href="https://developers.themoviedb.org/3"
        >
          API Docs
        </a>
      </li>
    </ul>
  </nav>
</header>
`;
